client/containers/Budget/index.js

```diff
@@ -25,9 +25,9 @@ class Budget extends Component {
     const data = { transactions, categories };
 
     return (
-      <div>
-        <BudgetGrid data={data} />
-      </div>
+      <Chunk load={() => import('components/BudgetGrid')}>
```

Comment:
@sulfurious I've been going by the linting rules we set up for this app (i.e. by running `npm run lint` from the command line: note that gives us two errors from prior commits, unrelated to this PR).